src/components/section/ArticlesSearch.jsx

```diff
@@ -0,0 +1,37 @@
+import NewsCard from "../ui/NewsCard";
+import { useSelector, useDispatch } from "react-redux";
+import { searchCategory, loadingData } from "../../features/news";
+import { useEffect } from "react";
+import { getEverything } from "../../features/api";
+const ArticlesSearch = () => {
+  const searchArticles = useSelector((state) => state.news.search);
+  const newsArticles = useSelector((state) => state.news.articles);
+  const inputValue = useSelector((state) => state.news.inputValue);
+  const dispatch = useDispatch();
+
+  const filterAllArticles = [
+    ...(searchArticles?.articles || []),
+    ...(newsArticles?.articles || []),
+  ]?.filter(
```

Comment:
Why filtering, if this the search functionality, it is not a good idea  becouse this will limit the search capability of you news app, instead, you can use (q) param in the api documentation to search in all articels and order only 10 articles by using pageSize param.